fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/Column.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Column.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2015-2022 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.apple.foundationdb.record.query.plan.temp;
+
+import com.apple.foundationdb.record.query.predicates.Value;
+
+import javax.annotation.Nonnull;
+import java.util.Objects;
+
+/**
+ * A case class to hold a values computing a column together with information about the field of a record it supplies
```

Comment:
Java doesn't have case classes. I believe the precedent for things that will be able to be `record` -- the closest thing -- in the future it to have a TODO comment to that effect.

Reply:
A `Column` is just a convenience class in order to avoid writing `Pair<Field, Value>` all the time. I removed the `case class` reference in the documentation.